Kajabi/sage-lib
# Form Select: React documentation update

## Description
Update Form Select documentation to better explain props in React.


## Screenshots

|  Before  |  After  |
|--------|--------|
| <img width="1252" alt="Screenshot 2023-10-11 at 3 57 48 PM" src="https://github.com/Kajabi/sage-lib/assets/24237393/d9aceeb7-0eaf-41e2-856d-e96e72e017ce"> | Coming soon... |

## Testing in `sage-lib`
1. Navigate to http://localhost:4100/?path=/docs/sage-select--default-story
2. Verify the documentation renders and is valid

## Testing in `kajabi-products`
1. (**LOW**) Documentation only update, nothing tested in kajabi-products
